com.unity.ml-agents/Editor/MLAgentsSettingsBuildProvider.cs

```diff
@@ -0,0 +1,74 @@
+using System.Linq;
```

Comment:
This might be a dumb question, but why do we have a BuildSettingsProvider if none of the settings are used in a build? (They are all Editor only settings?)

Reply:
You mean the use case it to remove the command line argument listeners on the build? Because a build will not try to listen to a trainer unless a specific command line argument is passed.

Reply:
Good question. Originally I made a player training port but that doesn't seem to make much sense so I removed that.
But I still think there's chance that we'll add settings that is used in player (analytics? academy stepping?)